lib/gui/app/modules/analytics.js

```diff
@@ -128,7 +128,12 @@ exports.logDebug = resinCorvus.logDebug
  *   image: '/dev/disk2'
  * });
  */
-exports.logEvent = resinCorvus.logEvent
+exports.logEvent = (message, data) => {
+  const sessionData = {
+    sample: mixpanelSample
+  }
+  resinCorvus.logEvent(message, { ...data, sessionData })
+}
```

Comment:
Do we need this extra `sessionData` layer? It will only contain `sample`.

Reply:
We don't... *for now*, so I guess it's fine to remove it.
*For now* 👀

Reply:
It seems like we do in order to use the spread operator

Reply:
We don't for now, added it to make it more clear that we can add w/e to that context.
We can put it back when/if needed